lib/sequel/model/exceptions.rb

```diff
@@ -24,11 +24,18 @@ def initialize(message=nil, model=nil)
   UndefinedAssociation = Class.new(Error)
   ).name
 
-  (
   # Raised when a mass assignment method is called in strict mode with either a restricted column
   # or a column without a setter method.
-  MassAssignmentRestriction = Class.new(Error)
-  ).name
+  class MassAssignmentRestriction < Error
+    # The Sequel::Model object related to this exception.
+    attr_reader :model
+
+    def initialize(msg, model)
```

Comment:
This is not backwards compatible, and will break code that uses `raise MassAssignmentRestriction`.  I recommend using a different class method, such as `create`.

Reply:
Maybe better to make model arg optional?

Reply:
I would prefer a separate method.